code/modules/surgery/organs/internal/eyes/_eyes.dm

```diff
@@ -51,26 +53,30 @@
 	var/native_fov = FOV_90_DEGREES
 
 /obj/item/organ/internal/eyes/Insert(mob/living/carbon/eye_recipient, special = FALSE, drop_if_replaced = FALSE)
+	if(ishuman(eye_recipient))
+		var/mob/living/carbon/human/human_recipient = eye_recipient
+		old_eye_color_left = human_recipient.eye_color_left
+		old_eye_color_right = human_recipient.eye_color_right
```

Comment:
no, do this on `on_insert()` instead. no reason to have code above the call to parent when the framework allows for us to do it another way